ext/exiv2/exiv2.cpp

```diff
@@ -1,4 +1,5 @@
 #include "exiv2/image.hpp"
+#include "exiv2/error.hpp"
```

Comment:
```suggestion
ifndef BOOST_PREPROCESSOR_DEBUG_ERROR_HPP
#include "exiv2/error.hpp"
endif 
```

Disclaimer: I have no idea what I'm doing

Reply:
@patrobinson can you try and compile this against your machine if you have `exiv2` v0.26 or earlier ?

Reply:
Sorry I don't have it installed and it doesn't appear to be available via brew

Reply:
Tried on v0.26 and it works fine! Will see if I can get an earlier version (0.24) as it's mentioned in the readme.

Reply:
0.24 also seems to have that header :)